README.md

```diff
@@ -1,28 +1,78 @@
 # github-action-detect-unused-secrets
 
-GitHub Action that automatically detects unused secrets comparing the workflow secret names with the repo secret names.
+GitHub Action that detects unused secrets.
+Compares the secret names in your repository with the secret names under `.github/workflows`
 
 ## Example usage
 
-```yaml
-name: detect-unused-secrets
+The example below runs the action every Monday at 10am.
+Generates a token with required permissions which `secrets.GITHUB_TOKEN` does not have.
+Uses checkout code to get access to the `$GITHUB_WORKSPACE`,
+And finally runs the `github-action-detect-unused-secrets` with the generated token.
 
-on: [push]
+```yml
+ name: detect-unused-secrets
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 10 * * MON'
 
 jobs:
   setup:
     runs-on: ubuntu-latest
     steps:
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@v1
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.PRIVATE_KEY }}
+      - name: Checkout Code
+      - uses: actions/checkout@v3
       - name: Detect unused secrets
-        uses: nearform/github-action-detect-unused-secrets@v1
+        uses: nearform/github-action-detect-unused-secrets@main
```

Comment:
in the docs let's show using a tagged version, not a branch. basically undo this change